articles/postgresql/flexible-server/concepts-elastic-clusters-limitations.yml

```diff
@@ -25,74 +25,79 @@ sections:
+
   - name: Extensions
     questions:
-      - question: Why is TimescaleDB not available with elastic clusters?
-        answer: TimescaleDB extension isn't supported on elastic cluster due to low-level conflicts with the Citus extension.
-      - question: What extensions aren't supported during preview?
-        answer: |
-          The following extensions aren't supported during the preview:
-            - anon
-            - pg_qs - Query Store
-            - postgis_topology
-            - TimescaleDB
+      - question: Which extensions are available for elastic clusters?
+        answer: Elastic clusters are compatible with the majority of PostgreSQL extensions.  For a comprehensive list, please refer to the [extension compatibility matrix](https://github.com/citusdata/citus/blob/main/EXTENSION_COMPATIBILITY.md) in the Citus GitHub repository.  Be sure to review the release notes and support details when using any of the available extensions alongside elastic clusters.
+
+      - question: Why isn't TimescaleDB available with elastic clusters?
+        answer: Elastic clusters make use of key low-level system hooks which conflict with the TimescaleDB extension.  Fortunately, elastic clusters provide similar time-series capabilities.  You can learn more about timeseries data in the [Citus documentation](https://docs.citusdata.com/en/stable/use_cases/timeseries.html).
 
   - name: Migrations
     questions:
       - question: How can I migrate to/from elastic clusters?
-        answer: Currently, migrations to/from elastic clusters can be done with pg_dump, pg_restore, and pgcopydb. Any other tool working with standard PostgreSQL should work.
+        answer: Migrations to/from elastic clusters can be done with pg_dump, pg_restore, and pgcopydb. Any other tool working with standard PostgreSQL should work.
+
+      - question: Can I convert my existing Public Preview instance to General Availability (GA) elastic clusters?
+        answer: If you have created any elastic clusters during the Public Preview phase, you are required to re-create them once the feature becomes Generally Available.  If this raises concerns, or if you need help determining whether your server meets the necessary requirements, please contact us by going to the Azure portal, selecting your Azure Postgres resource, and following the instructions in the banner.
+
   - name: Networking
     questions:
       - question: Can I use PgBouncer for connection pooling with elastic clusters?
-        answer: Yes, you can use PgBouncer with elastic clusters. Port 6432 should be used for schema and node management operations, while port 8432 is load-balanced to PgBouncer instances running across all nodes in the cluster.
+        answer: Yes, PgBouncer is supported with elastic clusters. Port 6432 should be used for schema and node management operations, while port 8432 is load-balanced to PgBouncer instances running across all nodes in the cluster.
```

Comment:
This part should be added in main article for connection mangement/ports too not just FAQ